src/services/posts/record-likes.test.js

```diff
@@ -0,0 +1,38 @@
+const {expect} = require('chai')
+const db = require("../../db")
+const recordPostLikes = require("./record-likes")
+const {fixtures} = require('../../../test/utils')
+
+
+describe('RECORD LIKES on gif', () => {
+	let user
+	let post
+	before(async ()=>{
+		 user = await fixtures.insertUser() 
+		post = await fixtures.insertPost({
+			userId : user.id , 
+			type : 'gif'
+		})
+	})
+	it('should throw an error if Gif is already liked', async () => {
```

Comment:
this scenario is to actually get the error thown if the Gif has already been liked

1. add an `insertPostLike` to the test/utils. it should follow the format we already have
2. in this test case, you call the `insertPostLike` so that it inserts the like into the db for that postId
3. And then you call the main function you're testing which is currently called `recordPostLikes`
4. It should throw an error. The expectation will look something like what is here: 
https://github.com/tbf-challenge/teamwork-be/blob/f793f00a53b88c7c8b8755a6852234df3a54f431/src/services/users/reset-password/index.test.js#L17-L22

Reply:
i have made the changes and pushed to github